.github/workflows/marathon_33.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow runs all the checks on pull requests
+
+name: Pull Request Checks
+
+on: pull_request
+
+jobs:
+  marathon_33:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Assemble android tests
+        run: ./gradlew -PCI=true assembleDebugAndroidTest assembleDebug
+      - name: Print
+        run: find -name "*.apk"
+      - name: run tests on android 33
+        uses: MarathonLabs/action-test@0.3.3
+        with:
+          apiKey: ${{ secrets.marathon_github_token }}
```

Comment:
Just FYI, secrets except `GITHUB_TOKEN` is not accessible for `pull_request` in case of PRs from forks